Extension/src/common.ts

```diff
@@ -776,6 +777,17 @@ export function promptReloadWindow(message: string): void {
     });
 }
 
+export function CreateTempFileWithPostfix(postfix: string): Promise<tmp.FileResult> {
```

Comment:
Sorry I missed this the first time.  The function should start with a lower case letter.